src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  --main-color: #00acdc;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  margin: 0 50px;
```

Comment:
Take a look at your solution – you have horizontal margins disallowing `nav` to spread all over screen width